### DISCUSSION TITLE
b-table sort

### DISCUSSION THREAD

**Discussion by shes6006** [bootstrap-vue/bootstrap-vue#7220] [Q&A]
I'm using b-table default sorting method and I found that in the method the all value will be transformed to String as follows: 
```
function toString(value) {
  if (value === null || typeof value === 'undefined') {
    return ''
  } else if (value instanceof Object) {
    return Object.keys(value)
      .sort()
      .map(key => toString(value[key]))
      .join(' ')
  } else {
    return String(value)
  }
}
```
therefore, when my field values are alphabets with numbers and starts with '0' the sorting will cause problem. 
For example, '007D2' will be smaller than '00500', cause 00500 will go to "String(value)" and I get "320"
I wonder if it is inevitable or the toString function above can be improved ? 

FYI, I customized my compare routine to avoid the situation, but I just wonder there is any solution to avoid it in the toString function

**Comment by mitulagrawal607** [comment]
Can you share your fix for this problem? Or did you get some solution for this ?
